### ISSUE TITLE
Fail 1.4.4 - Resize text

### ISSUE THREAD

**Issue by neilmispelaar** [DTS-STN/Life-Journeys#103]
### Description

WCAG [1.4.4 - Resize text](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-scale.html) requires that a page be able to increase the text size 200% without any clipping or obfuscation of the textual elements.

When testing the page there was some clipping. 

#### Firefox screenshot

<img width="1360" alt="Screen Shot 2021-08-08 at 9 33 33 PM" src="https://user-images.githubusercontent.com/9325038/128652517-5de97f44-f366-40ef-be8e-012e3f5907c4.png">

#### Safari screenshot

<img width="1360" alt="Screen Shot 2021-08-08 at 9 35 00 PM" src="https://user-images.githubusercontent.com/9325038/128652570-0dddb2e5-b846-4386-8956-eea49c75b888.png">

### How to test 

1. Visit the project page 
2. Set the zoom to "Zoom text only" (View -> Zoom -> Zoom text only)
3. Select Zoom In (View -> Zoom -> Zoom In) twice to get to 200% 
4. Scroll the page and look for any text over lapping / clipping / issues

<img width="1023" alt="Screen Shot 2021-08-08 at 9 36 37 PM" src="https://user-images.githubusercontent.com/9325038/128652670-7d0d4f5e-2eb0-4f70-b584-adc6ad1db1b0.png">

### Resources 

[W3C - SC 1.4.4 - Resize text](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-scale.html)
[F69: Failure of Success Criterion 1.4.4 when resizing visually rendered text up to 200 percent causes the text, image or controls to be clipped, truncated or obscured](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/F69)

**Comment by P0NDER0SA** [comment]
Thanks... we fixed this one already because of an issue identified by IITB A11y.  thanks
